src/daemon/collector/content/collect.js

```diff
@@ -100,20 +100,49 @@ const findMenu = () => {
 
 // FB5
 const hideMenu = () => {
-  for (let i=0; i<10; i++){
+  for (let i = 0; i < 10; i++) {
     setTimeout(function () {
-      const menus = document.querySelectorAll('[role="menu"]')
+      const menus = document.querySelectorAll('[role="menu"]');
+      const buttonsWithExpandableMenus = document.querySelectorAll(
+        '[aria-expanded="false"][aria-haspopup="menu"]'
+      );
+
       if (menus) {
-        for (let j=0; j<menus.length; j++) {
-          if (menus[j].clientHeight > 0 && !menus[j].closest('[data-pagelet="ChatTab"]') && menus[j].innerText.indexOf('Notfification') === -1) {
+        for (let j = 0; j < menus.length; j++) {
+          const menuNotExpanded = userHasNotExpandedMenu(
```

Comment:
Naming tips. Avoid negatives where possible. Ideally we lead with the value type or action, before we move to a higher contexts

`const isMenuExpanded = hasUserExpandedMenu(...)`

Great cheatsheet https://github.com/kettanaito/naming-cheatsheet